common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -1155,4 +1166,30 @@ private void verifyTokenParametersAreSupported(@NonNull final TokenCommandParame
                             + " Current required version is set to: " + parameters.getRequiredBrokerProtocolVersion());
         }
     }
+
+    /**
+     * Check if have received broker version not supported error.
+     * @param resultBundle Result bundle from a broker operation.
+     * @param requiredBrokerProtocolVersion Required broker protocol version sent in request.
+     * @throws UnsupportedBrokerException if result contains broker not supported error.
+     */
+    private void verifyBrokerVersionIsSupported(@Nullable final Bundle resultBundle, @Nullable final String requiredBrokerProtocolVersion) throws UnsupportedBrokerException {
+        final String methodTag = TAG + ":verifyBrokerVersionIsSupported";
+        if (resultBundle == null) {
+            Logger.info(methodTag, "result bundle is null");
```

Comment:
Is this an error case? Do we need to throw?

Reply:
This method is just to check if version check succeeds. Each operation later has validation to create success or failure results.  I added broker protocol version failure in same manner across all operation, so it can be verified at common place here.
So, essentially handling that error here while letting the existing code take care of per operation validation and result creation.

Reply:
Ok, "later" seems strange. To me it seems like the check for `resuldBundle == null` needs to be done outside this method (prior to calling this method), and then this method can make the param NonNull and not have to worry about the nullability of the param because that this is then already validated.

I'm not sure how big of a change that is, so I'm fine with what you have for now.

Reply:
Actually null check is there for each operation before this method is called.